src/app/calendar/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_library(calendar-lib calendar.cpp connector.cpp)
+
+target_compile_options(calendar-lib PRIVATE -Werror -Wall -Wextra -pedantic)
```

Comment:
You can add a 
target_link_libraries(clinic-lib PUBLIC doctor-lib rom-lib)
to remove `"../clinic/"` in calendar.cpp

Reply:
Added